doc/source/structures/celestial_bodies/atmosphere.rst

```diff
@@ -104,6 +97,23 @@ A Structure closely tied to :struct:`Body` A variable of type :struct:`Atmospher
 
     The altitude at which the atmosphere is "officially" advertised as ending. (actual ending value differs, see below).
 
+Deprecated Suffix
+-----------------
+
+.. attribute:: Atmosphere:SCALE
+
+    :type: :ref:`scalar <scalar>`
+    :access: Get only
+
+    A math constant plugged into a formula to find atmosphere density.
+
+    .. note::
+
+        .. deprecated:: 0.17.2
+
+            To get the same functionality, you must use :attr:`Vessel:PATCHES`  which is a suffix of the :struct:`Vessel` itself.
```

Comment:
Vessel:PATCHES has nothing to do with Atmosphere.  I'm assuming this text is some kind of a cut-n-paste from somewhere else that was supposed to get re-written here but never was.